blog-api-y-nguyen/Controllers/AuthorsController.cs

```diff
@@ -55,14 +59,14 @@ public IActionResult PutAuthor(int id, Author author)
             {
                return BadRequest();
             }
-            _authorRepository.PutAuthor(author);
+            _authorService.PutAuthor(author);
             try
```

Comment:
as I suggested, the `_authorService.Save()` is no need, should include it in the CRUD operation and throw error at the lowest level, the API would return `500 server error` in that case. Practically we need to handle that in middleware or something but in the scope of this project no need to do that